Bottom: 3d29ffacfb74b5d90fa9d14fbde5aac100ff3c54
Top:    fc97a6d53d8049162bb0a946bf49915bf58b01b1
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-28 15:42:06 +0800

Refresh of tun-rx-batching

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 030286c..6ea5d6d 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -75,7 +75,7 @@
 
 #include <linux/uaccess.h>
 
-static int rx_batched = 0;
+static int rx_batched;
 module_param(rx_batched, int, 0444);
 MODULE_PARM_DESC(rx_batched, "Number of packets batched in rx");
 
@@ -1328,8 +1328,9 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 		local_bh_disable();
 		netif_receive_skb(skb);
 		local_bh_enable();
-	} else
+	} else {
 		err = tun_rx_batched(tfile, skb, more);
+	}
 #else
 	netif_rx_ni(skb);
 #endif
